roles/variables/defaults/main.yml

```diff
@@ -203,7 +203,8 @@ confluent_cli_version: 3.63.0
 ### Recommended replication factor, defaults to 3. When splitting your cluster across 2 DCs with 4 or more Brokers, this should be increased to 4 to balance topic replicas.
 default_internal_replication_factor: 3
 
-### SASL Mechanism to set on all Kafka Listeners. Configures all components to use that mechanism for authentication. Possible options none, kerberos, plain, scram, scram256
+### SASL Mechanism to set on all Kafka Listeners. Configures all components to use that mechanism for authentication. Possible options none, kerberos, plain, scram, scram256.
+### You can provide a comma-separated list of mechanisms to configure multiple listeners with different mechanisms. For example, 'kerberos,plain'
 sasl_protocol: none
```

Comment:
shouldn't this now be a list ?

Reply:
it is still a string of comma separated values

Reply:
we cannot make it as list because we need to have backward compatibility as well